zone/mob.cpp

```diff
@@ -2920,6 +2946,62 @@ void Mob::SendAppearanceEffect(uint32 parm1, uint32 parm2, uint32 parm3, uint32
 	safe_delete(outapp);
 }
 
+void Mob::SetAppearenceEffects(int32 slot, int32 value) 
+{
+	for (int i = 0; i < MAX_APPEARANCE_EFFECTS; i++) {
```

Comment:
You’re using < max +1 in other places, not sure your intentions there or here.

Reply:
Fixed the place where it did +1, forgot to change it.

Reply:
also fixed the <= issue. Missed a few things when I rewrote it.